### PR TITLE
Randomizer fix

### PR DESCRIPTION
**Pull request by suyashmahar**
This PR includes:
* Addition of *Contributing* and *Credits* section to README.md
* Fix in predictable behavior of pseudo-random number generator 
* Change min level API from 19 to 16 
* Fix version number inconsistency for demo app

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+when we return.
+
+Security
+--------
+
+Please do not publish security vulnerabilities publicly until we've had a chance
+to address them. All security related issues/patches should be sent directly to
+[sdsmobilelabs@gmail.com](mailto:sdsmobilelabs@gmail.com) where we will attempt to address them quickly. If you're
+unsure whether something is a security issue or not, then please be cautious and
+contact us at Sdsmobilelabs@gmail.com first.
```
Comment (dakshitagrawal) [inline CONTRIBUTING.md]: sdsmobilelabs@gmail.com

Comment (suyashmahar) [reply CONTRIBUTING.md]: @dakshitagrawal97 Can you elaborate that a bit?

**Comment by kdexd** [comment]
:green_heart: :checkered_flag: for _CONTRIBUTING.md_. This is great @suyashmahar

File: CONTRIBUTING.md (hunk 1 above)
Comment (kdexd) [reply CONTRIBUTING.md]: he probably means that S shouldn't be uppercase.

Comment (suyashmahar) [reply CONTRIBUTING.md]: @karandesai-96 @dakshitagrawal97 I'll fix it, thank you!

BTW @karandesai-96 sab uthaya hua hai

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 
-<div style="text-align:center"><img src="resources/trianglify-logo-180.png" data-canonical-src="trianglify-logo-180.png" width="154" height="154" /></div>
+<div align="center"><img src="resources/trianglify-logo-180.png" data-canonical-src="trianglify-logo-180.png" width="154" height="154" /></div>
```
Comment (kdexd) [inline README.md:2]: How about a GIF with 6 images where color of one triangle changes between two images ? :stuck_out_tongue:

Comment (suyashmahar) [reply README.md:2]: @karandesai-96 I will try but I don't have skills or tools for designing

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+then please drop into our [MDG public chat room](https://mdg.sdslabs.co/chat).
+If no one is online then you can still leave a message that will hopefully get a reply
+when we return.
+
+Security
+--------
+
+Please do not publish security vulnerabilities publicly until we've had a chance
+to address them. All security related issues/patches should be sent directly to
+[sdsmobilelabs@gmail.com](mailto:sdsmobilelabs@gmail.com) where we will attempt to address them quickly. If you're
```
Comment (csoni111) [inline CONTRIBUTING.md:45]: Markdown automatically converts `emails` to `mailto:` links, so we probably do not need to explicitly specify this!

File: README.md (hunk 2 above)
Comment (csoni111) [reply README.md:2]: You don't need any tools for that. It's all available online, just find 6 images with different colors (I hope you can do that). Then you may try something like [gifmaker](http://gifmaker.me/) or [ezgif](https://ezgif.com/maker) etc.

File: CONTRIBUTING.md (hunk 3 above)
Comment (suyashmahar) [reply CONTRIBUTING.md:45]: @csoni111 Didn't knew this, I'll correct this. Thanks

File: README.md (hunk 2 above)
Comment (suyashmahar) [reply README.md:2]: @csoni111 I will do that

**Pull request merged**